python-type-hints-multiple-types/requirements.txt

```diff
@@ -0,0 +1,4 @@
+mypy==1.5.1
+mypy-extensions==1.0.0
+tomli==2.0.1
```

Comment:
I believe, we don't need this dependency, which is a backport of the standard library module `tomllib` introduced in  Python 3.11.